src/pages/tutorial/angular/step-1.mdx

```diff
@@ -0,0 +1,919 @@
+---
+title: 1. Installing Carbon
+tabs:
+  ['Overview', 'Step 1', 'Step 2', 'Step 3', 'Step 4', 'Step 5', 'Wrapping up']
+---
+
+import Preview from 'components/Preview';
+
+### Starting with the Carbon Angular, there are two ways to begin working with Carbon components. By the end, these two methods will produce the same result.
+
+<AnchorLinks>
+
+<AnchorLink>Prerequisites</AnchorLink>
+<AnchorLink>Use starter app - recommended</AnchorLink>
+<AnchorLink>Start from scratch</AnchorLink>
+
+</AnchorLinks>
+
+## Preview
+
+A [preview](https://angular-step-2-carbon-tutorial.netlify.com) of what you will build:
```

Comment:
https://angular-step-2-carbon-tutorial.netlify.com/repos hitting this URL directly does not work. You probably need to modify your Netlify settings for the SPA to redirect all routes to `/`.

For simplicity, this is why use used the hash router for the React tutorial so every route is just the index route. https://react-step-2--carbon-tutorial.netlify.com/#/repos

Reply:
Swapping to the hash router is definitely the most compatible fix ... we'll _probably_ want to do that too

Reply:
done